### ISSUE TITLE
Followers of the Apocalypse Overhaul

### ISSUE THREAD

**Issue by Kelprunner** [sunset-wasteland/sunset-wasteland#98]
If the title wasn't obvious, the FoA clinic and, frankly, the entire faction need an overhaul. Their lore, their equipment, the clinic... I'll be touching on the latter two, primarily, but I'll drop a few things that I think _must_ be done lore-wise to make them fit in and mesh better. Of course, I'm just a random cat in the grass, so, keep your salt shaker on hand.

In Fallout: New Vegas, the Followers were considered weak by most of the factions. And yet, they've been around for nearly a hundred and twenty years! A hundred and sixty, considering Sunset takes place forty years after F:NV. This is because they are _very_ smart and pragmatic people as a whole. They're not going to die over a patch of land when they can set up their labs and clinics elsewhere. They won't start a fight, but if you force one on them? They'll damn well _end_ it. Or just leave town. Whichever is better for everyone in the long-run, and is more likely to work.

If you're going to describe them as a 'pacifist,' it'd be best to do so while mentioning the volunteers for the World Wars. The folks who were against violence and yet, when called to serve, picked up arms and fought for their country. Of course, not all of them would kill, but _true_ pacifists would be hard to find in the Wasteland, I imagine. Still, it's an option for people who don't like fighting OOCly or just want to try a pacifist character in general. Anyway, that's the basic lore stuff... So now more lore stuff, but this time about guns! And tech.

The fact they're not a merchant force, and have to maintain the Boneyard after their relations soured with the NCR, means their stockpiles of equipment are strained... And, yet, we see them with some actually okay equipment! 10mm SMGs, Hunting Rifles, 12-gauge shotguns... And _experimental, pre-war combat implants._ The Followers, in a frontier outpost... _Are selling experimental, pre-war implants._ Now, it's not cheap. Said implants are abhorrently expensive at a few thousand caps minimum, but it's implied they're reproducing _incredibly_ advanced pre-war tech with that. And... selling them for willing test subjects? Maybe? They seem to be the only truly good faction out there, but in a setting like Fallout, _who knows._ Anyway, the point is, they're at _least_ on the same playing field as the Brotherhood of Steel, technologically. Only, instead of focusing on the scavenging of pre-war tech, they aim to understand and advance it so they might uplift the Wastes. Or, at least, make it so it will survive humanity shooting itself in the foot yet again.

Even as a frontier outpost relying on mercenary guards, they _really_ shouldn't be starting with no weapons, no armor, no money, and not even usable machines! While their place should be somewhat ramshackle considering how far out they are, it also shouldn't be **_borderline useless_** like it is now!

![followers01](https://user-images.githubusercontent.com/106592384/171163873-4ee1c1a2-e086-4bd4-84e3-49921d1e0594.PNG)

Just... Look at this atrocity. People hang out in that tiny lobby all the time, crowding it beyond belief. The reception area is horrifically cramped and the surgery area is so small you spend more time shuffling each other, accidentally breaking lights put in the worst possible place, and trying to throw out random idiots that come in to watch surgery than you do actually working. 

![followers02](https://user-images.githubusercontent.com/106592384/171164144-d3e8ff39-fa9d-4507-ae19-38663e106f73.PNG)

The growing area should, honestly, be a semi-public farm in Bighorn, the reception area needs to be shorter and wider, the recovery rooms should be removed (added to a 2nd floor above the main floor?) and the OR needs overhauling. Like, man, I'd take _Box_ medical over this!

![Followers03](https://user-images.githubusercontent.com/106592384/171164475-4667a9ed-60c5-49d9-94d7-3706974519df.PNG)

Now, to the basement. Locker room and prep area? They're fine. Though, for the locker room, maybe make it a _smidge_ larger and add a few regular, empty lockers. The green arrow is pointing to a not-pictured part of Bighorn's tunnel system. It'd be _amazing_ buuuuut it's not actually connected. It honestly looks like the clinic got slapped over a pre-existing structure. And the chem room... Why in the world are there unbuilt machines when the Followers neither have a circuit printer or spare chem boards?

![Followers04](https://user-images.githubusercontent.com/106592384/171164825-37c63d3f-9ba0-4820-9546-47f1f514076e.PNG)

Finally, we have the top half. The morgue is fine and the dorms are okay, if public. But, as a Follower, you're probably too busy to do anything there anyway, so... Eh. Anyway, lastly, you have the tech room. The R&D server, if it's not going to generate points, should just go. Instead, it should have - as I've so _wonderfully_ labeled - a destructive analyzer and a techfab pre-built. Those are a **must.** More optionally, I think the Followers could do with a Circuit Imprinter and a mechfab _but_ their R&D network should have a curated research list. Remove all the mechs - except the APLU? - and maybe even come up with some unique implants for them to print. Health regen in the sunlight? Taser-punches that run on nutrition? Though that's a discussion for another time.

Anyway, with all that said - they desperately need to be made part of the town in some manner. Preferably near the bar and with access to the ORMs underground. Why? People already use the clinic as a player hub, and the Khans already defend the clinic in lieu of a dedicated guard force. Considering how mellow the local Khans are, I'm sure there's an acceptable reason they'd take that deal. Plus, this means more people will actually use the town for socialization! Hopefully.

### TL; DR 

- Lore: The Followers aren't suicidal pacifists. They're logical pragmatists in a wild, wild Wasteland
- Lore: They are _not_ a charity, but they will offer aid at a great discount or accept unconventional payment for emergency aid (Eg- for revival surgery on someone poor)
- Give the FoA spawns better equipment (9mm Pistols / Wattz Pistols, survival knives)
- Add a FoA armory with T1 / 2 equipment (Hunting/Double Barrel shotguns, 10mm SMGs, Mk. 23(?) .45 pistols) in limited quantity. NO reloading bench!
- Fix the Clinic's layout so it's not a living hell just to use. I would unironically take Box med over this, please
- Update their R&D room to be actually functional and more lore-accurate besides
- Move the clinic to the town to promote it being used, the Khan/Town spawns as a whole, and to make the place simply make more sense logically speaking
- Make the farm shared with the town itself, but unable to be accessed by random Wasters

**Comment by ExoticJazz** [comment]
Most things have been addressed in PR #109 

Also due to the new location and further goals to ensure the Khans and Followers coexist, they will not get their own armory as of now, it may change in the future but the new location deeper within Khan territory should assist with protection greatly.

**Comment by ExoticJazz** [comment]
> Good to know! I'll have to see the new layout before I can really comment on things, but I'm hopeful. As an aside, shouldn't they still have an armory anyway - perhaps, explicitly _because_ they're in the heart of the Khans' territory? Idk any IC explanation why they're so closely tied together, but I'd imagine Follower administration being worried about the Khans trying to pressure the docs into making chems for them. Even if they are exceptionally friendly so far.
> 
> Of course, OOC, the one doc having a 10mm pistol or something wouldn't dissuade the usual 4 or 5 khans in full combat gear from doing it anyway, but, it's worth mentioning at least.

The lore is mentioned on our Discord in the #lore-n-guides channel, it specifically explains the bond between FOTA and Khans. And khans have their own chemists for focusing on drug/chem production.

Anywho, I'll be closing this for now as the new update is about to go live within the next few hours; any further recommendations should be condensed into a separate feature request after you experience the new changes.
